### DISCUSSION TITLE
Upgrading from 19.4.x to 20: themes broken

### DISCUSSION THREAD

**Discussion by f1-outsourcing** [OpenMage/magento-lts#3901] [Questions & Answers]
I have all kinds of missing javascript files, there are a few.

/skin/frontend/base/default/js/lib/selectivizr.js
/skin/frontend/base/default/js/lib/matchMedia.js
/skin/frontend/base/default/js/lib/matchMedia.addListener.js

 Is there something like a guide for migrating themes?

**Comment by MoT3rror** [comment]
Never seen an answer for this. I can ran into the same issue. I solved it by clearing the whole cache.
